### ISSUE TITLE
/usr/lib/tar: Permission denied

### ISSUE THREAD

**Issue by ShalokShalom** [ocaml/opam#5650]
1. I installed both [opam](https://gitlab.archlinux.org/archlinux/packaging/packages/opam/-/blob/91bcd57c725d52a533a8aaae13a382996685fad9/PKGBUILD) and [ocaml](https://gitlab.archlinux.org/archlinux/packaging/packages/ocaml/-/blob/0348558f69e34aca873f02e556b99d094f1a2298/PKGBUILD) on Arch via the official repositories. 

2. I run `opam init`

/usr/bin/opam: "create_process" failed on /usr/lib/tar: Permission denied

Additional info: 

```
╰─λ opam config report
# opam config report
# opam-version         2.1.5
# self-upgrade         no
# system               arch=x86_64 os=linux os-distribution=garuda os-version=soaring
[ERROR] Opam has not been initialised, please run `opam init'
# read-state           OpamStd.OpamSys.Exit(50)

```

**Comment by ShalokShalom** [comment]
I had my /usr/lib on PATH, removing that worked.

**Comment by rjbou** [comment]
It is the same issue than #5585 & #5587. A PR is open with a fix #5606.

**Comment by ShalokShalom** [comment]
Oh, thanks. And I thought it was only my fault 😋
